packages/validator/src/cli-validator/run-validator.js

```diff
@@ -34,6 +35,10 @@ const processInput = async function(program) {
     return Promise.reject(2);
   }
 
+  // Prototype:
+  // Set initial loglevel of the root logger to be 'info'.
+  log.setLevel('info');
```

Comment:
Isn't "info" too verbose to use it as default? Maybe "warning" would be better.

Reply:
When we implement this feature for real, we would probably want to use the root logger wherever we're currently using `console.log` now, and I think we'd want the root logger to remain at the "info" log-level so we can in fact call logger.info() to log "informational" messages to the console (in addition to warning and error messages which would use logger.warn() and logger.error(), respectively).

Reply:
Okay, that makes sense. Thanks!